lib/mix/test/mix/tasks/format_test.exs

```diff
@@ -523,6 +523,28 @@ defmodule Mix.Tasks.FormatTest do
     end)
   end
 
+  test "uses inputs and configuration from :relative_to path", context do
+    in_tmp(context.test, fn ->
+      File.write!(".formatter.exs", """
+      [
+        locals_without_parens: [foo: 1]
+      ]
+      """)
+
+      root = File.cwd!()
+      File.cd("lib")
```

Comment:
You need to revert that so it doesn't affect other tests.

Reply:
You can use `File.cd!/2` 🙃

Reply:
i had totally forgotten this one @whatyouhide thanks!